components/webpack/webpack.config.js

```diff
@@ -12,6 +12,7 @@ module.exports = (env, argv) => ({
     alias: {
       'bittorrent-tracker': path.resolve(__dirname, '../../node_modules/bittorrent-tracker'),
       'brave-ui': path.resolve(__dirname, '../../node_modules/brave-ui/src'),
+      '~brave-core-ui': path.resolve(__dirname, '../../ui/webui/components'),
```

Comment:
it seems to be the tilde is easy to miss and I'm afraid some people (me) would spend a good time figuring out what's wrong to realize it's missing. wdyt?

Reply:
That's a good point. I wanted to differentiate from `@` prefix, which is valid prefix on npm. But we can probably just remove this if I can implement the overall 2nd proposal that we both wanted, instead of using `ui/webui/components` dir

Reply:
And of course this doesn't even need to be here for this architecture. It was just convenience as find / replace would be very time-consuming if it had to figure out the correct number of `../../` to put in to each replacement...